app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectsListActivity.kt

```diff
@@ -22,12 +22,20 @@ import com.sdp13epfl2021.projmag.database.Utils
 import com.sdp13epfl2021.projmag.model.ProjectFilter
 
 class ProjectsListActivity : AppCompatActivity() {
+
     private lateinit var itemAdapter: ProjectAdapter
     private lateinit var recyclerView: RecyclerView
-
+    private val appliedProjects: MutableList<ProjectId> = ArrayList()
+    private lateinit var utils: Utils
+
+    private fun updateAppliedProjects() {
+        utils.userDataDatabase.getListOfAppliedToProjects({ list ->
+            appliedProjects.removeAll { true }
```

Comment:
Wouldn't it be better ?
```kotlin
mutableList.clear()
```

Reply:
ah yes definitely! I could not remember the function